### DISCUSSION TITLE
How to set Analyzers?

### DISCUSSION THREAD

**Discussion by borjafreire** [shinyorg/shiny#881] [Q&A] [answered]
Hi, 
I'm trying to install Shiny on my application (First time with Xamarin)  but i always getting  'ServiceProvider has not been setup - have you initialized the Platform provider using ShinyHost.Init?' on MainActivity.cs.

In a video i saw that you put all the code on "Analyzers" but on my project i dont have that option.

Can you help me with that?

**Comment by aritchie** [comment, marked as answer]
You have to follow the docs & samples.

https://github.com/shinyorg/samples
https://allanritchie.com/posts/shiny20
